### ISSUE TITLE
Security error with Firefox in Homepage

### ISSUE THREAD

**Issue by davide-casiraghi** [davide-casiraghi/ci-global-calendar#161]
**Describe the bug**
I get a security error when trying to connect to the https site :
Error Code : SSL_ERROR_NO_CYPHER_OVERLAP

**Yves Salmon-Legagneur** has reported this bug on 5/10/2019

**To Reproduce**
1. Go to the homepage
2. See error

**Expected behavior**
The HP open as aspected

**Desktop (please complete the following information):**
- OSX 10.6.8 (MAC - dualcore2 2.4GHz macbook pro)
- Firefox 48.0.2  
- My preferences for the browser is to alert about "suspicious" sites, and not allow third party cookies.

**Comment by davide-casiraghi** [comment]
I have spent a little time investigating and I see that is related with some specific versions of Firefox.

**Comment by davide-casiraghi** [comment]
I have MAC too and Firefox and on my system works well. So since I can't reproduce it's harder to fix.

Anyway have spent a little time investigating and I see that is related with some specific versions of Firefox.
And to solve I have to fix something not on the code but on the server, and I don't know if I have enough privileges to fix it.

When I will have some more time I will dive deeper and contact the hosting.

**Comment by davide-casiraghi** [comment]
My first advice is to update Firefox to the last version

**Comment by davide-casiraghi** [comment]
Hi again Yves,

Usually in web development we use to fix internet explorer bugs for old version because it come standard with windows installation and in many companies is not possible to update it.
For Firefox and Google Chrome instead the tests are done just on the latest versions, because if are installed on the computer is assumed that the user can update them.
So backward compatibility is not provided.

You wrote me that you have Firefox version 48.0.2
On my mac I have version 69.0

So yours looks like very outdated..

Is it possible for you to update Firefox?

Let me know
Davide

**Comment by davide-casiraghi** [comment]
Hello Davide,

Thank you for your answer. 
Since Apple hid personal data (i.e. the Library) starting with OSX 10.7 version, I decided to stay with 10.6(.8) ... and to get rid of my Apple computers. Since then, I am in this process, which takes a while (!) as I am so used to OSX shortcuts etc.

I guess you shouldn't worry about the error, which indeed comes from an outdated Firefox version. I opened the site on Xubuntu FIrefox, it works perfectly and if I find a bug I'll let you/Ecki know.

++ Yves
